packages/eas-cli/src/commands/analytics.ts

```diff
@@ -0,0 +1,25 @@
+import { Command } from '@oclif/command';
+
+import Log from '../log';
+import UserSettings from '../user/UserSettings';
+
+export default class AnalyticsView extends Command {
+  static description = 'view or change analytics settings';
+
+  static args = [{ name: 'STATUS', options: ['on', 'off'] }];
+
+  async run() {
+    const { STATUS: status } = this.parse(AnalyticsView).args;
+    if (status) {
+      await UserSettings.setAsync('amplitudeEnabled', status === 'on');
+      Log.withTick(`${status === 'on' ? 'Enabling' : 'Disabling'} analytics.`);
+    } else {
+      const amplitudeEnabled = await UserSettings.getAsync('amplitudeEnabled', null);
```

Comment:
```suggestion
      const amplitudeEnabled = await UserSettings.getAsync('amplitudeEnabled', true);
```
?

Reply:
by default, analytics are enabled, so unless value is explicitly set there we assume it's true